mne_bids/read.py

```diff
@@ -320,7 +320,12 @@ def get_head_mri_trans(bids_fname, bids_root):
     mri_landmarks = mri_landmarks * 1e-3
 
     # Get MEG landmarks from the raw file
-    raw = read_raw_bids(bids_fname, bids_root)
+    _, ext = _parse_ext(bids_fname)
+    if ext == '.fif':
+        extra_params = dict(allow_maxshield=True)
+    else:
+        extra_params = None
```

Comment:
any chance to cover these lines in a test?

Otherwise LGTM

Reply:
I don't know what you're talking about.

Reply:
There isn't a test which goes into L324-325. It means that we don't know if that code path will work

Reply:
on second thoughts, it will be very tricky to do this. Let's merge it as is